docs/source/posts/2024/2024-08-21-final-report-robin.rst

```diff
@@ -0,0 +1,305 @@
+
+- **Tests for all endpoints** (ongoing)
+    It's important to have tests for all endpoints. Testing includes the following:
+
+    * Check the endpoints with valid data to see the response. Validate the JSON format.
+    * Check the endpoints with incorrect schema and record the response.
+    * Test by adjusting parameters like KNN.
+
+- **X Bot** (Optional Goal, deferred for now)
+    I had a talk about this with my mentors. This can be done by plugging the LLM backend into an X bot frontend, but they suggested spending my time improving model accuracy rather than simply adding another frontend for the LLM application.
+
+
+Other Open Source tasks
+-----------------------
+
+GSoC isn't all about what I do with my project. It exists along with the 3 other cool projects my peers - `Wachiou <https://github.com/WassCodeur>`_, `Iñigo <https://github.com/itellaetxe>`_ and `Kaustav <https://github.com/deka27>`_   did. I learnt a lot through them reviewing my PRs and me reviewing their PRs. I attended all the weekly meetings of Wachiou to learn about his progress and to learn new stuff. He attended all my meetings too, which was awesome :)
+
+Contributions to FURY apart from the ones directly part of GSoC:
+    * https://github.com/fury-gl/fury/pull/862 - Rendering videos on a cube
+    * https://github.com/fury-gl/fury/pull/861 - docstring improvements
+    * https://github.com/fury-gl/fury/pull/891 - Codespell fix
+    * https://github.com/fury-gl/fury/pull/893 - .gitignore modification
+    * https://github.com/fury-gl/fury/issues/924 - Raised issue
+
+Contributions to other repositories during this time, due to GSoC work:
+    * https://github.com/langchain-ai/langchain/issues/23515 - Langchain issue raised
+    * https://github.com/github/docs/issues/34258 - GitHub issue raised
+    * https://github.com/github/docs/pull/34259 - PR for the raised GitHub issue
+    * https://github.com/orgs/community/discussions/136436 - GitHub feature request
+
+
+Acknowledgement
+---------------
+
+I am very thankful to my mentors `Serge Koudoro <https://github.com/skoudoro>`_ and `Mohamed Abouagour <https://github.com/m-agour>`_. They were awesome and provided me with a comfortable environment to work in. Also got to thank `Beleswar Prasad Padhi <https://www.linkedin.com/in/3v3ryone>`_ who gave me a very good introduction to opensource. The good thing about open source is I can still work on this (and other FURY projects) till I'm satisfied. I'm excited to continue contributing to the open source community.
+
+
+Timeline
+--------
+
+
+.. list-table:: GSoC 2024 Weekly Reports
+   :widths: 15 50 50
+   :header-rows: 1
+
+   * - Week
+     - Description
+     - Blog Post Link
+   * - Week 0
+     - Community Bonding!
+     - https://fury.gl/latest/posts/2024/2024-05-28-week-0-robin.html
```

Comment:
it would be cool if you could use hypertext links with clickable text for blog post links, like you did with the other links above.

Reply:
+1